packages/loopback/lib/context/binding.ts

```diff
@@ -5,21 +5,15 @@
 
 export class Binding {
   public value: any;
-  private _isDynamic: boolean = false;
+  public getValue: () => any = () => { throw new Error(`No value was configured for binding {this._key}.`); };
```

Comment:
Does TypeScript have arrow functions with different semantics than JS? If not, then `this._key` is going to give a useless value.